assets/css/sensei-course-theme/quiz.scss

```diff
@@ -61,13 +61,17 @@
 			box-shadow: 2px 10px 30px 5px rgba(#E2E4E7, 0.5);
 		}
 
-		&__left {
-			flex: 1;
-
+		#sensei-quiz-actions {
+			flex-direction: row-reverse;
```

Comment:
It sounds a little "hacky". I started a discussion (p1642194430020300-slack-C07418EJ0) to see if we can define a standard for that.

Also, I was checking that for mobile it's a little different from the design. Could you tweak it to be like the design in the different variations?

Reply:
> It sounds a little "hacky". I started a discussion (p1642194430020300-slack-C07418EJ0) to see if we can define a standard for that.

Got it. Will leave the current implementation as is until we confirm the correct behavior.
 
> Also, I was checking that for mobile it's a little different from the design. Could you tweak it to be like the design in the different variations?

Fixed. b3da657fe1fdd5a56142153608956fe4d97da326 But will probably need another update if the pagination rendering markup changes as per the above issue.